packages/provider/src/client.ts

```diff
@@ -135,13 +138,10 @@ export class SequenceClient {
   public readonly transport: ProviderTransport
 
   public readonly defaultEIP6492: boolean
+  public readonly projectAccessKey?: string
+  public readonly analytics?: Analytics
 
-  constructor(
-    transport: ProviderTransport | MuxTransportTemplate,
-    store: ItemStore,
-    options?: SequenceClientOptions | number,
-    public projectAccessKey?: string
-  ) {
+  constructor(transport: ProviderTransport | MuxTransportTemplate, store: ItemStore, options?: SequenceClientOptions) {
```

Comment:
If `options` is no longer accepted as a `number`, then there are code paths that are no longer required.